_data/strings.yml

```
@@ -19,3 +19,11 @@ locales:
     language_name: "中文"
     language: "官话"
     qmk_firmware: "QMK固件"
+  en: &DEFAULT_EL
```

Shouldn't this be `el`? 
```suggestion
  el: &DEFAULT_EL
```